.circleci/config.yml

```diff
@@ -7,7 +7,10 @@ httpbin_local: &httpbin_local
   image: kennethreitz/httpbin@sha256:2c7abc4803080c22928265744410173b6fea3b898872c01c5fd0f0f9df4a59fb
   name: httpbin.org
 test_runner: &test_runner
-  image: datadog/docker-library:ddtrace_py
+  image: docker.pkg.github.com/datadog/dd-trace-ci/ddtracepy-dev:1.0
```

Comment:
Just leaving a note since we wanted to update this: `dd-trace-ci/python-dev:latest`